docroot/themes/contrib/civictheme/civictheme_library/components/03-organisms/card-container/card-container.stories.js

```diff
@@ -57,6 +61,20 @@ export const CardContainer = (knobTab) => {
     modifier_class: text('Additional class', '', generalKnobTab),
   };
 
+  const cardType = radios(
+    'Card type',
+    {
+      'Navigation card': 'navigation-card',
+      'Promo card': 'promo-card',
+      'Publication card': 'publication-card',
+      'Service card': 'service-card',
+      'Subject card': 'subject-card',
+      'Random': 'random',
```

Comment:
please remove random and use `Promo card` as default

Reply:
@AlexSkrypnyk  Removed random.